doc/classes/@GDScript.xml

```diff
@@ -673,27 +760,48 @@
 			</return>
 			<description>
 				Return a random floating point value between 0 and 1.
+				[codeblock]
+				# prints 0.375671
+				print(randf())
+				[/codeblock]
 			</description>
 		</method>
 		<method name="randi">
 			<return type="int">
 			</return>
 			<description>
-				Return a random 32 bits integer value. To obtain a random value between 0 to N (where N is smaller than 2^32 - 1), you can use remainder. For example, to get a random integer between 0 and 19 inclusive, you can use randi() % 20.
+				Return a random 32 bits integer value. To obtain a random value between 0 to N (where N is smaller than 2^32 - 1), you can use remainder.
+				[codeblock]
+				print(randi() % 20)       # prints random number from 0 - 19
```

Comment:
More interesting than two examples of `randi() % N` would be an example of `randi() % (N - M) + M`, e.g.:
`print(randi() % (100 - 20) + 20)  # prints random number between 20 and 99`

(BTW "from 0 - 19" sounds weird, I'd expect either "between 0 and 19" or "within 0 - 19")